public/css/index.css

```diff
@@ -140,21 +277,58 @@
 }
 
 .faq-section {
-	background-color: rgb(242, 204, 181);
+	background-color: rgb(255, 225, 149);
 	padding: 20px 75px 0px 75px;
 }
 
-.questions h2 {
-	color: rgb(42, 45, 78);
-	font-size: 20px;
-	font-weight: bold;
-	display: inline-block;
+.q-a {
+	background-color: rgba(255, 255, 255, 0.3);
+	padding: 15px 15px 10px;
+	margin-bottom: 25px;
 }
 
-.questions p {
-	color: rgb(64, 105, 143);
-	text-align: justify;
-	font-size: 15px;
+.question-section {
+	background-color: rgba(255, 255, 255, 0.5);
+	border-left: 10px solid #ec3fa6;
+	padding-left: 15px;
+}
+
+.answer-section {
+	padding-top: 10px;
+	padding-left: 25px;
+}
```

Comment:
Another detail I missed: add 15px right padding to both of these sections to make the text wrap at a more comfortable distance from the edge of their boxes

Reply:
Fixed!